<html>

<head>
<title>Lawrence H. Uricchio's Research Page</title>
</head>

<body>
<img src="DSC01852.jpg" align=bottom width=240></a></p>
<h1>Lawrence H. Uricchio</h1>
<div style="margin-left: 3%">
<p> CEHG Postdoctoral Fellow <br>
    Rosenberg Lab <br>
    Department of Biology <br>
    Stanford University  <br> 
</div>
</p>

<h2> Welcome </h2>
<div style="margin-left: 2%">
<p> Welcome to my research page.  I am a postdoctoral fellow of the <a href="http://cehg.stanford.edu/">
     Center for Comptational, Evolutionary and Human Genomics</a>  in <a href="https://rosenberglab.stanford.edu">Noah Rosenberg's 
    lab</a> in the <a href="http://biology.stanford.edu/">Department of Biology</a> at Stanford University, 
    where I study the impact of evolutionary 
    forces on patterns of DNA sequence variation in human populations.  I develop computational tools 
    that are (I hope!) useful for other researchers interested in parametric models in evolutionary genetics.  I also have
    a strong interest in teaching, mentoring, science education policy, and curricular design. 
<br>
<br>
    Prior to arriving at Stanford, I was a graduate student with <a href="http://dbts.ucsf.edu/hernandez_lab">Ryan Hernandez</a> 
    at UCSF, a scientific programmer with <a href="http://www.stat.uchicago.edu/~nicolae">Dan Nicolae</a> and 
    <a href="http://ober.bsd.uchicago.edu">Carole Ober</a> at the University of Chicago, a research technician at UCLA with 
    <a href="http://www.hopkinsmedicine.org/dermatology/research/basic_science/miller_lab/">Lloyd Miller</a> 
    (now at Johns Hopkins University), and an undergraduate physics major studying with 
    <a href="http://www.people.carleton.edu/~apattana/">Arjendu Pattanayak</a> at Carleton College.   
</p>
</div>

<h2> Publications </h2>
<p>

<a href="http://www.ncbi.nlm.nih.gov/pubmed/25417809/">Population genetic simulations of complex phenotypes with implications 
for rare variant association tests.</a> <br>
LH Uricchio, R Torres, JS Witte, RD Hernandez. Genetic epidemiology 39. 2015
<br>
<br> 

<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012482/">Robust forward simulations of recurrent hitchhiking.</a><br>
LH Uricchio, RD Hernandez. Genetics 197. 2014
<br>
<br>

<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3794468/">Accurate imputation of rare and common variants in a 
founder population from a small number of sequenced individuals.</a><br>
LH Uricchio, JX Chong, KD Ross, C Ober, DL Nicolae.  Genetic Epidemiology 36. 2012
<br>
<br>

<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698246/">Population genetics of rare variants and complex diseases.</a> <br>
MC Maher*, LH Uricchio*, DG Torgerson, RD Hernandez. Human Heredity 74. 2012
<br> <i>*denotes equal contributions</i>
<br>
<br>

<a href="https://scholar.google.com/citations?hl=en&user=XyVUpZYAAAAJ">see google scholar for complete publication list</a>
</p>

<h2> Research Interests </h2>

<h3> Understanding how selection impacts genetic architecture and tests of association </h3>

<div style="margin-left: 2%">
<p> Natural selection acts to promote advantageous genetic variation and to constrain deleterious genetic variation.  For 
    example, it is now well established that mutations conferring the ability to metabolize lactose in adulthood
    were advantageous in the proto-European population thousands of years ago.  This makes very good sense as an evolutionary 
    story: adults who could metabolize lactose were able to benefit from the domestication of cows, since they could use milk
    as a highly reliable food source.  Lactose intolerant individuals would not have shared this advantage. 
    <br>
    <br>
    In contrast, it is reasonable to think that some genetic variants might be disadvantageous.  For example, let's consider
    a human disease that has a genetic risk component.  If this disease decreases the chance of 
    producing fertile offspring (say, by reducing the expected lifespan of an affected individual), then it is reasonable to 
    suppose that natural selection might keep alleles that increase disease risk at low frequency in the population.
    <br>
    <br>
    <!--<img src="power_AFR.jpg" align=left width=350></a></p>-->
    But this now introduces an interesting quandary. If natural selection acts to constrain genetic variation that contributes
    to disease, then why are there so many common, heritable human diseases, such as asthma, cardiovascular disease, and
    diabetes?  My work (and the work of many others in the field of human population genetics), has sought to address this
    question in a quantitative framework using population genetics theory.  There are many possible explanations, including
    <ul>
      <li>Natural selection does not actually act on disease variation for common human diseases.</li>
      <li>Heritable human diseases are not as heritable as we think they are (i.e., genetics plays a smaller role than 
          environment).</li>
      <li>Disease alleles are detrimental in the context of the disease, but advantageous for another reason (e.g., a disease 
          variant increased the rate of surviving childhood in ancestral human populations, but shortens the expected
          lifespan in adulthood). As a result, disease variants can rise to higher frequency because these effects
          (partially) cancel.  This is an example of pleiotropy.</li>
      <li>Disease variants are actually under very strong negative selection, and many different sites in the 3 billion 
          base pairs of the human genome confer disease risk.</li>
    </ul>
    To date, it's been very difficult to tease apart these possible explanations and make sense of how selection does (or does 
    not) act on common, heritable human diseases.  My work has mostly focused on the last two possibilities.  I have
    used theoretical population genetic models of natural selection and complex phenotypes to model recent (~100,000 yrs) 
    human evolutionary history.  Together with my PhD advisor Ryan Hernandez and our collaborator 
    <a href="http://wittelab.ucsf.edu">John Witte</a>, I showed that 
    state-of-the-art techniques for detecting risk loci in the genome 
    <a href="http://biorxiv.org/content/early/2015/03/01/015917">have lower power than one would hope</a>
    when selection acts most strongly on phenotypes.  Our work doesn't say anything about whether we expect selection to act on 
    phenotypes, just that it's difficult to find the causal genetic loci when selection is strong.  As a result, it is difficult 
    to quantitatively compare all the possible explanations above, because we lack the tools to definitively rule out a role
    for rare alleles under selection in human phenotypes.  However, I am very hopeful that the current generation of tools can 
    be modified and extended to improve their performance, at least for some disease models.
    <br>
    <!--In the figure embedded in this section, 
    we see that statistical power (the probability that we can detect a causal locus) 
    is a decreasing function of action of selection (represented by &rho;).  When  &rho; is 1, selection
    acts directly on the phenotype.  When  &rho; is 0, the action of selection and the effect on the phenotype are uncorrelated.
    The log10(x) curve shows a function that is commonly used to map allele frequencies to phenotypic effects when assessing
    the performance of association tests.
    <br>
    <br>-->

</p>
</div>
<h3> How does linked selection impact genetic variation? </h3>

<div style="margin-left: 2%">
<p>   
   The human genome consists of 23 chromosomes, each inherited in a single copy from each parent.  As a result, genetic
   sites that are on a single chromosome are "linked", meaning that they tend to be passed on to offspring together.  This
   immediately raises an important point about natural selection: if selection acts on a given genetic variant, then alleles
   that are nearby on the same chromosome will also be subject to the same selective force.  Over long periods
   of time, recombination (a genetic mechanism for patching paternal and maternal chromosomes together in gametes), will tend
   to swap alleles onto different chromosomes, ameliorating the effect of linked selection.
   <br>
   <br>
   Traditionally, most of our best theoretical tools for investigating selection have focused on modeling selection on individual
   sites (and ignoring linkage), or modeling simple linkage schemes in which natural selection follows very simple patterns.
   Some of my ongoing work seeks to extend these simple models by incorporating more complex models through the use
   of computer simulations.  I am generally interested in how linked selection (both of advantageous alleles and deleterious 
   alleles) may have impacted the human genome and the genomes of other organisms.  This work relies heavily on the ability
   to perform <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012482/">computationally efficient simulations of linked 
   selection.</a>

</p>
</div>
<h3> Jointly inferring selection and demography </h3>
<div style="margin-left: 2%">
When natural selection acts to constrain deleterious  genetic variation, it tends to cause an up-tick in the fraction
of alleles in a population that are very-rare.  As a result, if we were to sequence the genomes of a large number of humans,
we could test whether natural selection has acted on our genomes simply by counting the number of individuals that
carry each genetic variant in our dataset.  If we observe an excess of rare alleles relative to a very simple theoretical 
calculation, we can infer that selection has acted to shape patterns of diversity in humans.
<br>
<br>
Unfortunately, this very simple approach is not correct. Other (non-selective) evolutionary events can affect the 
proportion of sites in the genome that are present, especially changes in population size.  In particular, when populations
grow rapidly, they tend to accumulate more rare alleles.  Hence, it is challenging to disentangle how selection and demography
have both acted to affect human genetic variation.
<br>
<br>
Some of my ongoing work seeks to attack this problem by building inference tools using simulations and Approximate Bayesian
Computation, as well as extending the highly influential Poisson Random Field model.  
</div>


<h2> Software </h2>
<p>
  <a href="http://uricchio.github.io/sfs_coder/">
  sfs_coder: a python-based front end to the population-genetic simulation software sfs_code</a><br>
  <a href="http://sourceforge.net/projects/sfscode/files/sfs_coder/">sfs_coder source files here</a>
</p>

<h2> CV </h2>

<a href="CV.pdf">CV.pdf</a>

<h2> Contact </h2>

<p> uricchio at stanford dot edu </p>

<h2> </h2>



