
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<title>Uricchio lab</title>
<meta charset="UTF-8">
<style>
h1 {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 36px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 26.4px;
}
h2 {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 24px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 26.4px;
}
h3 {
        position: static;
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 16px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 5px;
}
a { 
    color: blue;
}
p {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 13px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 17px;
}
blockquote {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 21px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 30px;
}
pre {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 13px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 18.5714px;
}
ul {
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        font-size: 13px;
        font-style: normal;
        font-variant: normal;
        font-weight: 400;
        line-height: 17px
}
ol {
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        font-size: 15px;
        font-style: normal;
        font-variant: normal;
        font-weight: 400;
        line-height: 20px
}
li.p {
        margin-left: 0;
        margin-right: 10%;
        margin-top: .5em;
        margin-bottom: .5em;
}
#nav {
	width: 100%;
	float: left;
	font-size: 15px;
        margin: 0 0 3em 0;
	padding: 0;
	list-style: none;
	background-color: #f2f2f2;
	border-bottom: 1px solid #ccc; 
	border-top: 1px solid #ccc; }
#nav li {
	float: left; }
#nav li a {
	display: block;
	padding: 8px 15px;
	text-decoration: none;
	font-weight: bold;
	color: #069;
	border-right: 1px solid #ccc; }
#nav li a:hover {
	color: #c00;
	background-color: #fff; }

.main {
  float: left;
  margin-right: 260px; /* Same as the width of the sidenav */
  font-size: 28px; /* Increased text to enable scrolling */
  padding: 0px 10px;
}


</style>
</head>
<body bgcolor="#FFFFFF">
<BODY BGCOLOR="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
<!-- Top part with logo, title -->
<table style="border:5px solid #069;" cellpadding="7" cellspacing="10" width="760" align="left">
<tr height="90">
<td colspan="2" height="90">
<div align="left">
<table cellpadding="5" cellspacing="0">
  <tr>
    <td><font face="ARIAL,helvetica" size="5"><b><span class=bannerText>
            Uricchio lab @Tufts</span></b></font><img src="images/banner.png" class="image1" align="left"  height="80" HSPACE="15" VSPACE="8"/></td>
  </tr>
  </td>
</table>
<!-- Left side navigation bars -->
<tr>
<td style="border:2px solid gray; background-color:#e6f2ff;" >
<div align="left">
<table cellpadding="0" cellspacing="2" width="170">
 <a class="twitter-timeline" data-tweet-limit="5" href="https://twitter.com/LawUricchio?ref_src=twsrc%5Etfw">@LawUricchio</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
</table>

<!-- Main content -->
<td style="border:2px solid gray; background-color:#F5F5F5;" align="left" valign="top"><!-- Table across top with photo -->
  <table  cellpadding="4" cellspacing="2" align="left">

<ul id="nav" style="font-size: 13px; line-height: 17px">
    <li><a href="index.html">Home</a>
    <li><a href="research.html">Research projects</a>
    <li><a href="pubs.html">Publications</a>
    <li><a href="teaching.html">Teaching</a>
    <li><a href="people.html">People</a>
    <li><a href="CV.pdf">CV</a>
    <li><a href="https://github.com/uricchio">GitHub</a>
    <li><a href="https://twitter.com/LawUricchio">twitter</a>
</ul>
</table>     
 
<div  id="research"> 
    <h2 style="clear: both; margin-left: 1%">The big picture</h2>
    <div style="margin-left: 2%"> 
     <p> Many species worldwide are experiencing rapid, human-induced changes in their environment that threaten their 
        persistence, but we still struggle to predict which species will adapt to these changing circumstances, and 
        how adaptation may affect species composition in nature. 
        We develop evolutionary models and statistical inference 
        techniques to detect selection signals in genomic and phenotypic data.  Long-term goals of our work are to
        use these models to predict when species maintain sufficient genetic variation to adapt to rapid environmental change,
        and to understand how evolutionary processes affect our ability to detect the genetic variation underlying heritable
        traits.

    </p>
    </div>
<br>

<h3 style="margin-left: 1%">When does adaptation occur in response to environmental change?</h3>

<div style="margin-left: 2%">
<p>   
    <img src="images/method.jpeg" align=right width=300  HSPACE="25" VSPACE="10"/>
   <br>
   Evolutionary biologists have catalogued many recent adaptation events across species. But most of inferences of adaptation
   proceed backwards in time -- we use a combination of genomic, phenptypic and ecological data to understand how evolution may have favored particular genetic variants in 
   particular environments.  But how can we translate these methods to understand when adaptation is likely to occur? Is adaptation always inherently unpredictable, or 
   are there certain traits and environmental changes that result in predictable outcomes? 
   <br>
   <br>
   We are developing new methods to infer adaptation rate and strength from genomic data along with my colleague <a href="https://enardlab.com/"> 
   David Enard </a>. Most classic methods to infer adaptation rate and strength have assumed that beneficial mutations provide large fitness advantages, but
   we are increasingly aware that adaptation can sometimes proceed through <a href="https://uricchio.github.io/papers/Uricchio2019EvoComp.pdf">polygenic adaptation</a> and/or selection on standing variation. The
   extent to which these processes occur and contribute to species persistence is an open question.
   <a href="https://uricchio.github.io/papers/uricchio2019NatEcoEvo.pdf">Our work</a> 
   argues that weakly beneficial alleles are likely to be a major contributor to the adaptation process. If we are able to learn more about the specific traits and genetic architectures
   that are likely be evolvable through polygenic adaptation or selection on standing variation, we may be able to improve our ability to predict when species will (or will not) adapt
   to their changing environment. 
   This work relies heavily on our ability 
   to perform <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012482/">computationally efficient simulations of linked 
   selection</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/25417809">complex demography</a>.
   <br>
   <br>
   <br>
</p>
</div>
   <p>
<h3 style="margin-left: 1%"> Predicting the outcome of competition in plant communities</h3>

<div style="margin-left: 2%">
<p>
<img src="images/jasp_ridge.jpg" align=left width=250  HSPACE="25" VSPACE="10"/>
Exotic species can dramatically affect species composition  by directly out-competing native species, but in some cases native and introduced species may coexist stably.
<a href="https://uricchio.github.io/papers/Uricchio2019AmNat.pdf">We studied competition between native and exotic species in California grasslands</a>. Our analysis suggests
that invaders dominate over California native species, but that the likelihood of persistence of natives depends integrally on the order of arrival of their exotic competitors.
Our results provide a contrast to many earlier studies that focused on undisturbed communities and found stabilizing niche differences between species, perhaps due to the long co-evolutionary history of naturally co-occurring species.  Our ongoing work seeks to incorporate genomic data into ecological models to predict when species composition is likely to depend on rapid evolutionary processes.
<p>
<br>
<br>
</div>

 <!---   <h3 style="margin-left: 1%"> How does natural selection affect association studies?</h3>
    <div style="margin-left: 2%">
    <p>
    <img src="images/RNA_exp.jpg" align=left width=300  HSPACE="25" VSPACE="10"/>

    Many traits have a genetic component. In humans, examples include eye color, height, and body mass index, and also many common diseases, such as asthma and cardiovascular disease.
    Evolutionarily, this poses a question: if some diseases have a genetic component, how can they be common? Put another way, why hasn't natural selection constrained disease-causing
    genetic variation to be very rare, thereby making diseases very rare?  
   
    Some hypotheses include:
    </p>
    <ul "margin-left: 5%">

      <li class="p">Natural selection did not act on disease variation for common human diseases in ancestral human populations (perhaps because these diseases are primarily a result of modern 
                    enviornments).</li>
      <li class="p">Disease genotypes are detrimental in the context of the disease, but advantageous for another reason (e.g., a disease
          variant increased the rate of surviving childhood in ancestral human populations, but shortens the expected
          lifespan in adulthood).
      <li  class="p">Disease variation is under very strong negative selection and most disease variation is very rare, but there are so many sites in the 3 billion
          base pairs of the human genome that confer disease risk that disease is still common (i.e., there are many different ways to get same the same disease with different rare
          genetic variants).</li>
      <li  class="p">Heritable human diseases are not as heritable as we think they are (i.e., genetics plays a smaller role than environment).</li>
    </ul>
    <p>
    To date, it's been very difficult to tease apart these possible explanations and make sense of how selection does (or does
    not) act on common, heritable human diseases.

    Our work has mostly focused on the first three possibilities.  We
    use population genetic models of natural selection and complex phenotypes to model recent (~100,000 yrs)
    human evolutionary history, and to infer how <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/evl3.97">natural selection shapes genetic variation</a>.
    During my PhD, we showed that
    state-of-the-art techniques for detecting risk loci in the genome
    <a href="http://genome.cshlp.org/content/26/7/863">are not as effective as one would hope</a>
    when selection acts directly on trait-related variation.  We also showed that selection on RNA expression is likely
    to <a href="https://uricchio.github.io/papers/Hernandez2019.pdf">dramatically increase the relative importance of rare variation in driving gene expression</a>, suggesting that
    evolutionary constraint has slowed the rate of evolution of genetic elements underlying gene expression.
    </div> -->


    <br>
    <br>
</p>
</div>

<hr>

</td>
</tr>
</table>

</body>
</html>
