<!DOCTYPE html>
<html>
<title>Research page for Lawrence H. Uricchio, PhD</title>
<meta charset="UTF-8">
<style>
h1 {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 36px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 26.4px;
}
h2 {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 24px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 26.4px;
}
h3 {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 16px;
	font-style: normal;
	font-variant: normal;
	font-weight: 500;
	line-height: 15.4px;
}
a { 
    color: blue;
}
p {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 15px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 20px;
}
blockquote {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 21px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 30px;
}
pre {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
	font-size: 13px;
	font-style: normal;
	font-variant: normal;
	font-weight: 400;
	line-height: 18.5714px;
}
ul {
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        font-size: 15px;
        font-style: normal;
        font-variant: normal;
        font-weight: 400;
        line-height: 20px
}
ol {
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        font-size: 15px;
        font-style: normal;
        font-variant: normal;
        font-weight: 400;
        line-height: 20px
}
li.p {
        margin-left: 0;
        margin-right: 10%;
        margin-top: .5em;
        margin-bottom: .5em;
}
#nav {
	width: 100%;
	float: left;
	margin: 0 0 3em 0;
	padding: 0;
	list-style: none;
	background-color: #f2f2f2;
	border-bottom: 1px solid #ccc; 
	border-top: 1px solid #ccc; }
#nav li {
	float: left; }
#nav li a {
	display: block;
	padding: 8px 15px;
	text-decoration: none;
	font-weight: bold;
	color: #069;
	border-right: 1px solid #ccc; }
#nav li a:hover {
	color: #c00;
	background-color: #fff; }
</style>
<h1>Lawrence H. Uricchio</h1>
<img src="images/banner.png" class="image1" align="left"  width="1200" HSPACE="25" VSPACE="10"/>

<ul id="nav">
    <li><a href="index.html">About me</a>
    <li><a href="research.html">Research projects</a>
    <li><a href="pubs.html">Publications</a>
    <li><a href="teaching.html">Teaching</a>
    <li><a href="collaborators.html">Collaborators</a>
    <li><a href="index.html#contact">Contact</a>
    <li><a href="https://scholar.google.com/citations?user=XyVUpZYAAAAJ&hl=en">Google scholar</a>
    <li><a href="CV.pdf">CV</a>
    <li><a href="https://github.com/uricchio">GitHub</a>
</ul>

<div id="research">  
  <h2 style="clear: both;" align="left">Research Projects</h2>
   <h3 style ="margin-left: 1%">The big picture: how do natural selection and competition shape biological diversity?</h3>
    <div style="margin-left: 2%">
    <p> 
    All living things must compete to survive and produce offspring. Sometimes competition occurs between members of the same species (for example, competition for 
    mates) or between members of different species (for example, different species of grass competing for space and sunlight).  Competition
    should usually reduce diversity, because it removes individuals from the population or causes some individuals to have fewer offspring than others.  Given that
    competition is likely to be ubiquitous, this raises an important question in evolution and ecology: why do we observe such striking diversity among living things?
    On the evolutionary side, my research seeks to address this question using models of natural selection and large genomic datasets, while on the ecological side
    I use models of infectious disease and competition between species to make predicitions about diversity in interacting communities.
    </p>
    </div>
<h3 style="margin-left: 1%"> Predicting the outcome of competition and pathogen sharing in plant communities</h3>

<div style="margin-left: 2%">
<p>
<img src="images/ILS.png" align=left width=300  HSPACE="25" VSPACE="10"/>
Together with my postdoctoral co-advisor Erin Mordecai, I am using data from competition experiments between five species of grasses (two native to California and three exotics) to characterize how recent invasions impact native grass species in California. Exotic species can dramatically change species composition by directly out-competing native species, but in some cases native and introduced species may coexist stably, for example if each species occupies a unique niche. Pathogen spillover between species can also impact coexistence, and can either promote coexistence or competitive exclusion. I developed Markov Chain Monte Carlo methods to infer the strength of competitive interactions between each of these five species of grasses, and to understand the fitness cost of fungal infection. <a href="https://www.biorxiv.org/content/early/2018/01/24/253518">Our results </a> show that despite strong competitive signatures that decrease the fitness of the invading species, invaders still dominate over California native species and are likely to exclude California natives in the absence of other ecological forces promoting coexistence. The transmission of fungal infection has an overall weak impact on plant fitness in this system and does not substantially increase the chance that California native species will persist. Our results provide a contrast to many earlier studies that focused on undisturbed communities and found stabilizing niche differences, and suggest that creative management strategies may be required to prevent extinction of endemic plant life in this grassland community.
<p>
<br>
</div>

   <h3 style="margin-left: 1%"> How does natural selection affect genetically inherited traits?</h3>
    <div style="margin-left: 2%">
    <p>
    <img src="images/SFS_hutt.png" align=right width=300  HSPACE="25" VSPACE="10"/>

    Many traits have a genetic component.  In humans, examples include eye color, height, and body mass index, and also many common diseases, such as asthma and cardiovascular disease.  
    Evolutionarily, this poses a question: if some diseases have a genetic component, how can they be common?  Put another way, why hasn't natural selection constrained disease-causing
    genetic variation to be very rare, thereby making diseases very rare?  The short answer is: although we have many promising leads, we're still working on figuring it out.  Some
    hypotheses include:
    </p>
    <ul "margin-left: 5%">
         
      <li class="p">Natural selection did not act on disease variation for common human diseases in ancestral human populations.</li>
      <li class="p">Disease genotypes are detrimental in the context of the disease, but advantageous for another reason (e.g., a disease 
          variant increased the rate of surviving childhood in ancestral human populations, but shortens the expected
          lifespan in adulthood).
      <li  class="p">Disease variation is under very strong negative selection and most disease variation is very rare, but there are so many sites in the 3 billion 
          base pairs of the human genome that confer disease risk that disease is still common (i.e., there are many different ways to get same the same disease with different rare
          geneteic variants).</li>
      <li  class="p">Heritable human diseases are not as heritable as we think they are (i.e., genetics plays a smaller role than environment).</li>
    </ul>
    <p>
    To date, it's been very difficult to tease apart these possible explanations and make sense of how selection does (or does 
    not) act on common, heritable human diseases.  My work has mostly focused on the first three possibilities.  I
    use population genetic models of natural selection and complex phenotypes to model recent (~100,000 yrs) 
    human evolutionary history, and make inferences and predictions about how <a href="https://www.biorxiv.org/content/early/2017/08/25/173815">natural selection shapes genetic varition</a>.  
    Together with my PhD advisor Ryan Hernandez and our collaborators at UCSF
    <a href="http://wittelab.ucsf.edu">John Witte</a>, <a href="http://zaitlenlab.ucsf.edu/">Noah Zaitlen</a>, and <a href="http://yelab.ucsf.edu/">Jimmie Ye</a>, I showed that 
    state-of-the-art techniques for detecting risk loci in the genome 
    <a href="http://genome.cshlp.org/content/26/7/863">are not as effective as one would hope</a>
    when selection acts most strongly on phenotypes.  We also showed that selection on RNA expression is likely 
    to <a href="https://www.biorxiv.org/content/early/2017/12/15/219238">dramatically increase the relative importance of rare variation in driving gene expression</a>, suggesting that
    evolutionary constraint has controlled the rate of evolutiuon of genetic elements underlying gene experession.
    <br>
    <br>
</p>
</div>

<h3 style="margin-left: 1%"> How does linked selection impact genetic variation? </h3>

<div style="margin-left: 2%">
<p>   
    <img src="images/Vpsi.png" align=left width=300  HSPACE="25" VSPACE="10"/>
   The human genome consists of 23 chromosomes, each inherited in a single copy from each parent.  As a result, genetic
   sites that are on a single chromosome are "linked", meaning that they tend to be passed on to offspring together.  This
   immediately raises an important point about natural selection: if selection acts on a given genetic variant, then alleles
   that are nearby on the same chromosome will also be subject to the same selective force.  Over long periods
   of time, recombination (a genetic mechanism for patching paternal and maternal chromosomes together in gametes), will tend
   to swap alleles onto different chromosomes, ameliorating the effect of linked selection.
   <br>
   <br>
   
   Traditionally, most of our best theoretical tools for investigating selection have focused on modeling selection on individual
   sites (and ignoring linkage), or modeling simple linkage schemes in which natural selection follows very simple patterns.
   Some of my ongoing work seeks to extend these simple models by incorporating more complex models through the use
   of computer simulations.  I am generally interested in how linked selection (both of advantageous alleles and deleterious 
   alleles) may have impacted the human genome and the genomes of other organisms.  <a href="https://scholar.google.com/citations?user=pG3iI1wAAAAJ&hl=en"> 
   David Enard </a> and I are working to 
   understand these models both theoretically and empirically in the human genome. This work relies heavily on the ability
   to perform <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012482/">computationally efficient simulations of linked 
   selection</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/25417809">complex demography</a>.

</p>
</div>
<hr>

